src/components/common/YourCompany/YourCompany.vue

```diff
@@ -238,8 +238,10 @@
       <v-divider class="mx-4 my-1" />
 
       <!-- Nature of Business -->
-      <div class="section-container">
-        <NatureOfBusiness />
+      <div id='nature-of-business' class="section-container" :class="{'invalid-section': invalidNatureOfBusiness}">
+        <NatureOfBusiness
+          :invalidSection="invalidNatureOfBusiness"
+        />
```

Comment:
This can be simplified, eg:

https://github.com/bcgov/business-create-ui/blob/main/src/views/Registration/RegistrationDefineBusiness.vue#L18

Also please use double quotes in the template.

Reply:
1. Ditto for office addresses below.

2. You can specify `class="section-container"` on the component as well.

Up to you on whatever looks cleaner.

Reply:
Looks good, yes. I think the ID in the page is probably overwriting the ID in the shared component. Probably still OK though.